### PR TITLE
adding notifications for all project features

### PR DESCRIPTION
**Pull request by Bankacha**
adding notifications for all project features

### CODE REVIEW

File: src/admin/components/AddToSupply.js [hunk 1]
```diff
@@ -8,12 +8,12 @@ import React from "react";
 export function AddToSupply(props) {
 
     const dispatch = useDispatch()
-    const [quantity, setQuantity] = useState(0);
+    const [quantity, setQuantity] = useState('');
```
Comment (stveljko012) [inline src/admin/components/AddToSupply.js:12]: number?

File: src/notifications/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { store } from 'react-notifications-component';
```
Comment (stveljko012) [inline src/notifications/index.js:1]: Bravo!

**Review by stveljko012 — APPROVED** [review]
good job!

File: src/notifications/index.js (hunk 2 above)
Comment (Bankacha) [reply src/notifications/index.js:1]: Hvala :)

**Pull request merged**